.github/workflows/integration-test-call.yml

```diff
@@ -1,77 +0,0 @@
-name: "Integration test (using workflow call)"
```

Comment:
Do we not want to keep testing the existing MRVA flow? This is still used by our beta testers, so we still shouldn't break this until everyone is actually using live results.

Reply:
You're right we should keep testing it. I was a bit too hasty in trying to delete it.

I'd prefer not to compromise on the new approach too much trying to fit it in with the old approach, but that's a separate matter. If we're ok with having semi-duplicate checks on PRs for the next couple of months then that's fine.

Reply:
I've added the old tests back, or rather I've rebased this PR so it no longer removes the old tests. The first commit of this PR is the same as before but it no longer removes the old test code. Then the other commits after the first one are addressing comments or making changes.